myget tenant service tests failing

:clock11: **When did the problem start?**

~ a week ago

:camera: **Live badge**

https://img.shields.io/cefsharp.myget/cefsharp/v/cef.sdk
![](https://img.shields.io/cefsharp.myget/cefsharp/v/cef.sdk)

:wrench: **Is the live badge working?**

All the others are. I think we just need a new example for the test.

:link: **CircleCI link**

https://app.circleci.com/pipelines/github/badges/daily-tests/1158/workflows/924ad2a8-a60b-466e-8f60-36f844223da9/jobs/2059/tests

:beetle: **Stack trace**

```
AssertionError: label mismatch: expected 'myget' to equal 'cefsharp'
    at Function._expectField (file:///home/circleci/project/shields/core/service-test-runner/icedfrisby-shields.js:70:51)
    at IcedFrisbyNock.<anonymous> (file:///home/circleci/project/shields/core/service-test-runner/icedfrisby-shields.js:54:26)
    at IcedFrisbyNock.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:954:10)
    at invokeNextHook (node_modules/icedfrisby/lib/icedfrisby.js:1003:24)
    at /home/circleci/project/shields/node_modules/icedfrisby/lib/icedfrisby.js:1017:7
    at new Promise (<anonymous>)
    at IcedFrisbyNock._runHooks (node_modules/icedfrisby/lib/icedfrisby.js:976:12)
    at IcedFrisbyNock.run (node_modules/icedfrisby/lib/icedfrisby.js:1276:20)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Context.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:1348:9)
```

:bulb: **Possible solution**

As I say, I think we just need a new example project for this test case, but I don't know how to find one. Anyone familiar with MyGet able to help?

--- a/services/myget/myget.service.js
+++ b/services/myget/myget.service.js
@@ -25,11 +25,11 @@ class MyGetVersionService extends Version {
       title: 'MyGet tenant',
       pattern: ':tenant.myget/:feed/v/:packageName',
       namedParams: {
-        tenant: 'cefsharp',
-        feed: 'cefsharp',
-        packageName: 'cef.sdk',
+        tenant: 'tizen',
+        feed: 'dotnet',
+        packageName: 'Tizen.NET',
       },
-      staticPreview: this.render({ version: '91.1.1' }),
+      staticPreview: this.render({ version: '9.0.0.16564' }),
     },
   ]
 }
@@ -48,7 +48,7 @@ class MyGetDownloadService extends Downloads {
       namedParams: {
         tenant: 'cefsharp',
         feed: 'cefsharp',
-        packageName: 'CefSharp',
+        packageName: 'CefSharp.Common',
       },
       staticPreview: this.render({ downloads: 9748 }),
     },

--- a/services/myget/myget.tester.js
+++ b/services/myget/myget.tester.js
@@ -69,9 +69,9 @@ t.create('version (valid)')
   })
 
 t.create('version (tenant)')
-  .get('/cefsharp.myget/cefsharp/v/cef.sdk.json')
+  .get('/tizen.myget/dotnet/v/Tizen.NET.json')
   .expectBadge({
-    label: 'cefsharp',
+    label: 'dotnet',
     message: isVPlusDottedVersionNClausesWithOptionalSuffix,
   })
 

